### ISSUE TITLE
[linter] wrong indentation reported when using multi line properties

### ISSUE THREAD

**Issue by Cactusbone** [stylus/stylelint-stylus#22]
example inspired from https://stackoverflow.com/a/48514878/422995

case A
```
div
    box-shadow 0 0 0 black \
               1 0 0 black \
               0 0 1 black
```

aligning "columns" make it easier to read, however the stylus/indentation rule reports an error.

case B
```
div
    box-shadow \
    0 0 0 black \
    1 0 0 black \
    0 0 1 black
```
this passes the indentation rule, however it's harder to read

case C
```
div
    box-shadow \
        0 0 0 black \
        1 0 0 black \
        0 0 1 black
```
this looks okay to me, but does not pass the indentation rule

I'm not sure what the rule should be, but I think something like: "indented from the base rule" and "all line with the same indentation", without checking for how many indentation.

these could be configurable though.

In IntelliJ, the rules are named "Align when multiline", "Continuation indent" and "Align <something>" (here it could be "Align parameters when multiline" for case A)
